Makefile

```diff
@@ -42,6 +42,9 @@ install: $(INSTBIN) $(INSTMAN1)
 lint: $(BIN).c
 	clang-tidy $< -- $(CFLAGS)
 
+format: $(BIN).c
+	clang-format -i $<
```

Comment:
Could be good to add this to CI if more people are working on this in the future but not an issue for now